docs/index.rst

```diff
@@ -110,6 +110,8 @@
   logging-metric
   logging-sink
   logging-handlers
+  logging-handlers-GAEHandler
+  logging-handlers-GKEHandler
```

Comment:
These doc names are not like the others. Is this a problem? I don't know, but what you have against:

```
logging-handlers-gae-handler
logging-handlers-gke-handler
```